src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/manuell/brevmottaker/domene/ManuellBrevmottaker.kt

```diff
@@ -39,5 +39,19 @@ data class ManuellBrevmottaker(
                 landkode.isNullOrBlank()
         )
 
+    fun harGyldigAdresse(): Boolean =
+        if (landkode == "NO") {
+            navn.isNotEmpty() &&
+                !adresselinje1.isNullOrEmpty()
```

Comment:
```suggestion
                !adresselinje1.isNullOrEmpty() && 
```

Det mangler && her.

Nå skjer dette:

```
        if (landkode == "NO") {
            navn.isNotEmpty() &&
                !adresselinje1.isNullOrEmpty()

            !postnummer.isNullOrEmpty() &&
                !poststed.isNullOrEmpty()
        } 
```

Da valideres bare de to nederste

Reply:
Hoi, der har noe blitt fjernet som ikke skulle det ja. Jeg la til `&&` i en egen commit slik at jeg også kunne kjøre ktlint 😊